app/assets/javascripts/application.js

```diff
@@ -10,6 +10,7 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+//= require jquery
```

Comment:
needed this for jquery to work, we had a slim version and I needed the more robust version